video/out/wayland_common.c

```diff
@@ -1027,21 +1027,23 @@ static const struct wp_presentation_feedback_listener feedback_listener = {
 
 static const struct wl_callback_listener frame_listener;
 
+void vo_wayland_commit_surface(struct vo_wayland_state *wl){
+    if (wl->presentation) {
+        wl->feedback = wp_presentation_feedback(wl->presentation, wl->surface);
+        wp_presentation_feedback_add_listener(wl->feedback, &feedback_listener, wl);
+    }
+    wl->frame_callback = wl_surface_frame(wl->surface);
+    wl_callback_add_listener(wl->frame_callback, &frame_listener, wl);
+    wl_surface_commit(wl->surface);
+}
```

Comment:
This would break presentation/frame callback on opengl/vulkan. You *shouldn't* need to mess with this I hope. Is the presentation stuff somehow breaking the VO?